chapter02/sxg/src/main/java/com/github/sebin/chapter02/AccountTransaction.java

```diff
@@ -0,0 +1,50 @@
+package com.github.sebin.chapter02;
+
+import java.util.Calendar;
+
+public class AccountTransaction {
+	private int fromAccountId;
+	private int toAccountId;
+	private int amount;
+	private Calendar time;
```

Comment:
Nice, I totally forgot about the timestamp, Yes I believe we do need a timestamp for the transaction.